.github/workflows/stale.yml

```diff
@@ -7,12 +7,12 @@ jobs:
   stale:
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions/stale@v3
+    - uses: actions/stale@v9
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 30 days'
         days-before-stale: 30
         days-before-close: 30
         close-issue-message: 'Issue closed due to inactivity.'
         stale-issue-label: 'triage/stale'
-        exempt-issue-labels: 'triage/accepted,triage/discuss,kind/design-issue,kind/health,kind/roadmap-item,kind/task'
```

Comment:
why is there a diff here? as far as I can see there are no changes to this line.

Reply:
vim by default fixes up missing newline at the end of file.  Proper text files should end in a newline,
otherwise `cat f1.txt f2.txt` will have strange result.

```
$ git diff -U0 HEAD^2
diff --git a/.github/workflows/stale.yml b/.github/workflows/stale.yml
index c840eb08..dd7cb263 100644
--- a/.github/workflows/stale.yml
+++ b/.github/workflows/stale.yml
@@ -10 +10 @@ jobs:
-    - uses: actions/stale@v3
+    - uses: actions/stale@v9
@@ -18 +18 @@ jobs:
-        exempt-issue-labels: 'triage/accepted,triage/discuss,kind/design-issue,kind/health,kind/roadmap-item,kind/task'
\ No newline at end of file
+        exempt-issue-labels: 'triage/accepted,triage/discuss,kind/design-issue,kind/health,kind/roadmap-item,kind/task'
```